x-pack/plugins/security_solution/public/flyout/document_details/right/hooks/use_show_event_overview.ts

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { useMemo } from 'react';
+import type { EcsSecurityExtension as Ecs } from '@kbn/securitysolution-ecs';
+import { getField, getFieldArray } from '../../shared/utils';
+import { useRightPanelContext } from '../context';
+import type { GetFieldsData } from '../../../../common/hooks/use_get_fields_data';
+import { getRowRenderer } from '../../../../timelines/components/timeline/body/renderers/get_row_renderer';
+import { defaultRowRenderers } from '../../../../timelines/components/timeline/body/renderers';
+import { isEcsAllowedValue } from '../utils/event_utils';
+
+/**
+ * Returns true if the event.kind is ecs compliant or event renderer exists
+ */
+export const showEventOverview = (getFieldsData: GetFieldsData, dataAsNestedObject: Ecs) => {
```

Comment:
any specific reason for extracting this outside of the `useShowEventOverview` hook below?